### DISCUSSION TITLE
Notifications with tables in the html not sending

### DISCUSSION THREAD

**Discussion by MariskaMB** [solspace/craft-freeform#1029] [Questions & Answers]
**Describe the bug or issue you're experiencing**

When using a notification template with table elements in de html, the notification does not get sent. No error is logged either, so tables may not be the only problem. I managed to get the notification to be sent only by removing all tables in the template, however.

**Steps to reproduce**

1. Go the the notifications tab to edit a notification template.
2. In the body of the e-mail, add a `<table>` element.
3. Go to a form and add a notification using the template with the table in it.
4. Fill out the form on the website like a normal visitor and submit, triggering the notifications to be sent.

**Expected behavior**

Notifications being sent with all the usual html elements, or alternatively an error being logged.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.7.1
- Freeform Version: 5.0.6.1
- Freeform Edition: Pro
- Fresh Install or Upgrade: Upgrade

**Comment by kjmartens** [comment]
Hi @MariskaMB,

I have played around with this on my end and I cannot duplicate this issue. Are you 100% sure there weren't any errors in the template code? Can you try re-adding a table in the notification template and see if it fails? And if so, can you copy and paste the template code here? 🙂

**Comment by wolispace** [comment]
I had a similar situation, 

I am trying to output the table of data users can enter (using the Add+ button to add new rows).

The output of field.valueAsString is a comma-delimited list of all of the cells with no formatting.

I use [Insert_AI_code_generator_of_choice] to convert the field.value into table as that is what the user entered.

 It came up with this for the admin template - I have not touched the user template as yet:

```
{% extends "mail/_entry.twig" %}
{% block content %}
    <h2 style="line-height:1.25">New submission from your {{ form.name }} form</h2>
    <p>Submitted on: {{ dateCreated|date("l, F j, Y \\a\\t g:ia") }}</p>

    <table cellpadding="0" cellspacing="0" border="0" style="width: 100%;">
        <tbody>
            {% for field in allFields %}
                <tr>
                    <td style="padding: 5px; font-weight: bold; vertical-align: top;">{{ field.label }}</td>
                    <td style="padding: 5px;">
                        {% if field.value is iterable %}
                            <table cellpadding="0" cellspacing="0" border="0" style="width: 100%;">
                                <tbody>
                                    {% for row in field.value %}
                                        <tr>
                                            {% if row is iterable %}
                                                {% for cell in row %}
                                                    <td style="padding: 3px;">{{ cell }}</td>
                                                {% endfor %}
                                            {% else %}
                                                <td style="padding: 3px;">{{ row }}</td>
                                            {% endif %}
                                        </tr>
                                    {% endfor %}
                                </tbody>
                            </table>
                        {% else %}
                            {{ field.valueAsString }}
                        {% endif %}
                    </td>
                </tr>
            {% endfor %}
        </tbody>
    </table>
{% endblock %}

```

Before I added the code starting `{% if row is iterable %}` it sends admin and user notifications fine. 

With the above admin template it sends the user email fine, but not the admin email, and there are no visible errors that I can see.

if `{% if row is iterable %}` is not valid, would you expect it to fail with an error we see somewhere?

Bonus point: how do we output an html table of a field.value ? I see no reference to this in the documentation.

Thanks
